Jenkinsfile

```diff
@@ -67,18 +81,65 @@ node('mac-mini-ios') {
                     timeout(10) {
                         sh '''#!/bin/bash -l
                             set -e
-                            xcodebuild -workspace Client.xcworkspace -scheme "Fennec" -sdk iphonesimulator -destination "platform=iOS Simulator,OS=11.2,id=185B34BB-DCB8-4A17-BDCA-843086B67193" ONLY_ACTIVE_ARCH=NO -derivedDataPath clean build
+                            xcodebuild -workspace Client.xcworkspace -scheme "Fennec" -sdk iphonesimulator -destination "platform=iOS Simulator,OS=11.2,id=185B34BB-DCB8-4A17-BDCA-843086B67193" ONLY_ACTIVE_ARCH=NO -derivedDataPath clean build test
                         '''
                     }
                 }
+                stage('Setup Test Environment'){
+                    sh '''#!/bin/bash -l
+                        set -e
+                        npm install -g appium
```

Comment:
we could add those to `package.json` and run with `npm run` - this will add a better version control and it would pollute a global namespace

Reply:
Done.